tests/test_fdb.py

```diff
@@ -56,6 +61,20 @@ def test_getnextpdu(self):
         self.assertEqual(value0.type_, ValueType.INTEGER)
         self.assertEqual(value0.data, 113)
 
+    def test_getnextpdu_lag(self):
```

Comment:
can we add same case for for multi-asic also . `tests/namespace/test_fdb.py`

Reply:
we currently skip the tests/namespace/test_fdb.py as fdb MIB is not modified to support for mulit-asic as it is a T0 topology feature.

Reply:
ok ideally we should remove test itself then.